packages/commonwealth/client/scripts/views/components/component_kit/CWImageInput/UploadControl.tsx

```diff
@@ -203,6 +203,14 @@ export const UploadControl = ({
         <div
           className="generate-image-section"
           onClick={(e) => e.stopPropagation()}
+          onKeyDown={(e) => {
+            if (e.key === 'Enter' && imagePrompt.trim()) {
+              e.preventDefault();
```

Comment:
lets move `e.preventDefault();` before the if block and also add `e.stopPropogation()`, to always block keydown events from propagating to `CWForm`